src/app/header/header.component.scss

```diff
@@ -36,32 +29,41 @@
             margin-top: em(-3);
         }
     }
-    .navbar-collapse span.fa,span.pficon {
-        font-size:em(20);
-    }
-}
-@media (max-width: $grid-float-breakpoint){
-    .navbar-fixed-top {
-        position: fixed;
-        top: auto;
-    }
+     .navbar-collapse span.fa,span.pficon {
+         font-size:em(20);
+     }
+ }
+  @media (max-width: $grid-float-breakpoint){
     .navbar-collapse {
         max-height: none;
     }
-    body {
-        margin: 0;
-    }
     .nav-mob-scroll{
         overflow-x: scroll !important;
         .nav-list-mob-scroll{
             width: 170%;
         }
     }
-}
+ }
 .navbar-pf {
+    .navbar-primary
+    {
+      position:relative;
+      height:auto !important;
+      //min-height:$navbar-height
+    }
     .navbar-primary.persistent-secondary
-    >li.active {
-        margin-bottom: 0 ;
+    >li
+    {
+      a{
+        // hack:ensures elements do not cause secondsry menu to  'jiggle'
+        max-height:$navbar-height;
+      }
+      &.active {
+        &.with-no-children
+        {
+          margin-bottom:0px
```

Comment:
0px shows intent making it more clear what the author had in mind. it could be em rem of % , but pixels give fine grained control